client/app/telecom/pack/xdsl/modem/pack-xdsl-modem.js

```diff
@@ -87,6 +87,11 @@ angular.module('managerApp').config(($stateProvider) => {
         controller: 'XdslModemServiceCtrl',
         controllerAs: '$ctrl',
       },
+      'templatesView@telecom.pack.xdsl.modem': {
```

Comment:
Looks like this could be a component instead :)

Reply:
done